charts/orchestration/values.yaml

```diff
@@ -0,0 +1,9 @@
+replicaCount: 1
+
+image:
+  repository: ghcr.io/cdcgov/phdi/orchestration
+  pullPolicy: IfNotPresent
+  tag: "v1.0.7"
+
+service:
+  port: 8080
```

Comment:
You'll also need to add environment variables for the URLs of the other services that the orchestration service depends on (validation, fhir conversion, ingestion, and message parsing). The ingestion chart is a good example of how to do this - you'll need to add the [(placeholder) values](https://github.com/CDCgov/phdi-charts/blob/main/charts/ingestion/values.yaml#L13) here, and update the [deployment YAML](https://github.com/CDCgov/phdi-charts/blob/main/charts/ingestion/templates/deployment.yaml#L32) to read the values in and set them as environment variables. That way the Docker container can read the URLs as needed.

It may be a good idea to populate these values as secrets (which the ingestion service is [also doing](https://github.com/CDCgov/phdi-charts/blob/main/charts/ingestion/templates/secrets.yaml#L7).) It's not the most sensitive of data, but I could see a scenario where we're getting a lot of spam traffic if we don't obscure the URLs at least a little.